src/content/guides/compatible-terminals.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Compatible Terminals
+description: Compatible payment terminals.
+nav:
+  path: Reference/Merchant Integrations
+  order: 1
```

Comment:
@Wulingoh Do you need to bump the order for all the other guides under Merchant Integrations down too?

Reply:
It is done